contracts/adapters/GuildKick.sol

```diff
@@ -78,20 +87,20 @@ contract GuildKickContract is IGuildKick, DaoConstants, MemberGuard {
         return proposalId;
     }
 
-    function kick(DaoRegistry dao, uint64 proposalId)
+    function guildKick(DaoRegistry dao, uint64 proposalId)
         external
         override
         onlyMember(dao)
     {
-        GuildKick storage guildKick = kicks[proposalId];
-        // If status is empty or DONE we expect it to fail
+        GuildKick storage kick = kicks[proposalId];
+        // If it does not exist or is not in progress we expect it to fail
         require(
-            guildKick.exists && guildKick.status == GuildKickStatus.IN_PROGRESS,
+            kick.exists && kick.status == GuildKickStatus.IN_PROGRESS,
```

Comment:
if you already manage with status, you don't really need exists no ?  Especially since you always check exists & the status. That shows that the status is enough

Reply:
Yes. I was missing the default value for the `Status` enum. Just removed that flag. It is fine now. Thanks for flagging.